product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx

```diff
@@ -23,10 +25,16 @@ For example, to create a primary/standby high-availability cluster:
 ```shell
 biganimal cluster create
 __OUTPUT__
-Cluster architecture: High Availability
+Cluster architecture: Primary/Standby High Availability
 Number of standby replicas: 2 Replicas
 Enable read-only workloads: No
 Provider: Azure
+Cloud provider subscription ID:
+- 123123123
+- 456456456
+Service account IDs:
+- service-account-1234567b@development-data-123456.iam.gserviceaccount.com
+- service-account-1234567b@development-data-123456.iam.gserviceaccount.com
```

Comment:
- input displayed from user is incorrect
- interactive mode only numbers separated by comma is a valid input 
- service a/c id prompt is incorrect

*kindly refer the confluence page spec diff or maybe please run the CLI binary in the interactive mode to understand the context